### ISSUE TITLE
Environment ignores custom poolsize

### ISSUE THREAD

**Issue by erwinvaneyk** [fission/fission#891]
At least for sizes < 3, it seems that the `poolsize` param is completely ignored.

In the logs below I expected to find 1 instance of the pod. Instead 3 instances of Python have been deployed.

```
.03:15:19 ~$ fission env create --name python --image fission/python-env:0.10.0 --period 5 --mincpu 300 --poolsize 1 --maxcpu 300 --minmemory 256 --maxmemory 256
environment 'python' created

.03:16:34 ~$ fission env list
NAME   UID                                  IMAGE                     POOLSIZE MINCPU MAXCPU MINMEMORY MAXMEMORY EXTNET GRACETIME
python e8103e17-b5c4-11e8-9376-42010a9a0ff1 fission/python-env:0.10.0 1        300m   300m   256Mi     256Mi     false  5
.03:16:47 ~$ kubectl -n fission-function get po
NAME                                                              READY     STATUS    RESTARTS   AGE
python-e8103e17-b5c4-11e8-9376-42010a9a0ff1-udq6sxyi-5559fb6m7m   2/2       Running   0          19s
python-e8103e17-b5c4-11e8-9376-42010a9a0ff1-udq6sxyi-5559fnhbp2   2/2       Running   0          19s
python-e8103e17-b5c4-11e8-9376-42010a9a0ff1-udq6sxyi-5559fs2jhp   2/2       Running   0          19s

.03:17:02 ~$ fission -v
client:
  fission/core: {}
  generate-workflows:
    Version: 0.1.0
  gke:
    Version: 1.0.0
  workflows:
    Version: 75c8e3a498cb5f1928275ab901031f74baaf05b9
server:
  fission/core:
    BuildDate: 2018-08-18T07:41:18Z
    GitCommit: 95eef49cf8032dd9f9a2edcb985f91a7b1978e17
    Version: 0.10.0
```

**Comment by life1347** [comment]
You need to use `--version 3`. This is the issue we discussed at the meeting and you can see more discussion at https://github.com/fission/fission/issues/506 .

**Comment by erwinvaneyk** [comment]
Thanks @life1347 - then let me rephrase the issue to: it is completely unclear to a user when to use which version. As indeed discussed in the meeting, we need a way to avoid putting the burden of these kind options on the user.

**Comment by soamvasani** [comment]
Yeah, this is definitely a matter of using the right env version.  And yes, @erwinvaneyk, the UX here is terrible, but let's track that improvement in a separate issue.

@thenamly thanks for bumping that -- i've moved #758 up in priority.
